compair/static/modules/rich-content/rich-content-directive.js

```diff
@@ -251,6 +257,11 @@ module.directive('richContent',
                 };
 
                 $scope.showAttachmentContentModal = function (content) {
+                    // Close inline display before opening the modal dialog.
+                    // Avoid showing the same attachment with different orientation.
+                    if (content.displayInline) {
+                        $scope.toggleShowContent(content);
```

Comment:
I don't think we should force close this for the user

Reply:
Removed the auto-close logic